### PR TITLE
Update HelloWorld and Tx Signing test to use the latest Lit SDK

### PR DESCRIPTION
**Pull request by johnnynanjiang**
Problem

I got the following error when running `yarn run sign`

```
lit-protocol-js-serverless-function-test/node_modules/@lit-protocol/uint8arrays/src/lib/uint8arrays.js:15
        let charCode = str.charCodeAt(i);
                           ^

TypeError: str.charCodeAt is not a function
    at utf8Encode (/Users/nanjiang/Documents/dev/projects/Startup/J-Labs/projects/MPC/code/lit-protocol-js-serverless-function-test/node_modules/@lit-protocol/uint8arrays/src/lib/uint8arrays.js:15:28)
    at uint8arrayFromString (/Users/nanjiang/Documents/dev/projects/Startup/J-Labs/projects/MPC/code/lit-protocol-js-serverless-function-test/node_modules/@lit-protocol/uint8arrays/src/lib/uint8arrays.js:104:20)
    at LitNodeClientNodeJs.getLitActionRequestBody (/Users/nanjiang/Documents/dev/projects/Startup/J-Labs/projects/MPC/code/lit-protocol-js-serverless-function-test/node_modules/@lit-protocol/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.js:96:76)
    at LitNodeClientNodeJs.executeJs (/Users/nanjiang/Documents/dev/projects/Startup/J-Labs/projects/MPC/code/lit-protocol-js-serverless-function-test/node_modules/@lit-protocol/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.js:1088:38)
    at go (file:///Users/nanjiang/Documents/dev/projects/Startup/J-Labs/projects/MPC/code/lit-protocol-js-serverless-function-test/js-sdkTests/signTxn.js:34:39)
```

### CODE REVIEW

File: signTxnTest.js [hunk 1]
```diff
@@ -7,16 +7,16 @@ console.log("running!");
...
-  "pkpEthAddress": "0x2a5A2A9558118388e8f4bd1e1c32ac520CA7D0F4",
-  "pkpPublicKey": "0x04437f854b66f369fd70b5ec50fed7e1cf03d7d393c5f491b18dc74996ea646931b9d581c8dba213cfc28e806d83eedad313e364e2c9c760ee1eb9263b00e7b36a"
+  "pkpEthAddress": "0xfF8A9000a9866561dad7596f3342Fec18e8130d1",
+  "pkpPublicKey": "0x04cb229885d49c49559c3a45c91adae12ecf5b578412d666991a6f31b647b96327b82f719aa755e08c0432d25a234774908bb795c06778aa4b5900d21a7f4a7495"
 };
 
 const toAddress = "0x535b0dABaF59c90EeeBEf272b5F778C5369a1445"; // J Labs test account
 const chainId = 80001; // Polygon Mumbai Testnet
-const publicKey = pkp.pkpPublicKey;
+const publicKey = pkp.pkpEthAddress;
```
Comment (glitch003) [inline signTxnTest.js:16]: let's fix this variable name.  if it holds an address, it should not be called "publicKey"

File: js-sdkTests/signTxn.js [hunk 2]
```diff
@@ -1,27 +1,17 @@
...
-// NOTE: to replace with a new one that you get from oauth-pkp-signup-example
-const authSig = {
-  "sig": "0xe5e1d290ed645fd45d558dc8de176fc808dffb4bdc1c1c24e1cdc15f38f30a72251ef9c1e48bfd95568818d97894aff5ae1e2cc9da30c317e54262074f21823b1b",
-  "derivedVia": "web3.eth.personal.sign via Lit PKP",
-  "signedMessage": "localhost:3000 wants you to sign in with your Ethereum account:\n0x2a5A2A9558118388e8f4bd1e1c32ac520CA7D0F4\n\nLit Protocol PKP session signature\n\nURI: lit:session:9e0525c8caa0e70f85b829677f5abab27ae70344679ab67d9eda90c10b3160e2\nVersion: 1\nChain ID: 1\nNonce: 9OtojzBuln2qoTiH2\nIssued At: 2022-12-26T00:46:24.514Z\nExpiration Time: 2024-12-25T00:46:23.433Z\nResources:\n- urn:recap:lit:session:eyJkZWYiOlsibGl0RW5jcnlwdGlvbkNvbmRpdGlvbiJdfQ==",
-  "address": "0x2a5A2A9558118388e8f4bd1e1c32ac520CA7D0F4"
-};
+// cid for the bundled `build/singTxnTest.js`
+// to update run `yarn build`, upload to ipfs, and replace this cid.
+let ipfsCID = "Qmf9kGzSrNSki5DvgWs4G8FFyHMKhK7jJqHRptGosQo9NJ";
```
Comment (glitch003) [inline js-sdkTests/signTxn.js:14]: why are we replacing the JS in the file with an IPFS CID?  the purpose of this repo is to demonstrate how it all works, and we have specific tests for IPFS CIDs.  I think we should keep the JS in this test.

File: js-sdkTests/signTxn.js [hunk 3]
```diff
@@ -31,8 +21,19 @@ const go = async () => {
     litNetwork: "serrano",
   });
   await litNodeClient.connect();
+  
+  // you need an AuthSig to auth with the nodes
+  // normally you would obtain an AuthSig by calling LitJsSdk.checkAndSignAuthMessage({chain})
+  // NOTE: to replace with a new one that you get from oauth-pkp-signup-example
+  const authSig = {
```
Comment (glitch003) [inline js-sdkTests/signTxn.js:28]: Why are we changing the AuthSig and taking out SIWE resources, etc?

File: js-sdkTests/signTxn.js [hunk 4]
```diff
@@ -1,27 +1,17 @@
...
-// you need an AuthSig to auth with the nodes
-// normally you would obtain an AuthSig by calling LitJsSdk.checkAndSignAuthMessage({chain})
-// NOTE: to replace with a new one that you get from oauth-pkp-signup-example
-const authSig = {
-  "sig": "0xe5e1d290ed645fd45d558dc8de176fc808dffb4bdc1c1c24e1cdc15f38f30a72251ef9c1e48bfd95568818d97894aff5ae1e2cc9da30c317e54262074f21823b1b",
-  "derivedVia": "web3.eth.personal.sign via Lit PKP",
-  "signedMessage": "localhost:3000 wants you to sign in with your Ethereum account:\n0x2a5A2A9558118388e8f4bd1e1c32ac520CA7D0F4\n\nLit Protocol PKP session signature\n\nURI: lit:session:9e0525c8caa0e70f85b829677f5abab27ae70344679ab67d9eda90c10b3160e2\nVersion: 1\nChain ID: 1\nNonce: 9OtojzBuln2qoTiH2\nIssued At: 2022-12-26T00:46:24.514Z\nExpiration Time: 2024-12-25T00:46:23.433Z\nResources:\n- urn:recap:lit:session:eyJkZWYiOlsibGl0RW5jcnlwdGlvbkNvbmRpdGlvbiJdfQ==",
-  "address": "0x2a5A2A9558118388e8f4bd1e1c32ac520CA7D0F4"
-};
+// cid for the bundled `build/singTxnTest.js`
```
Comment (glitch003) [inline js-sdkTests/signTxn.js:12]: spelling error

File: js-sdkTests/signTxn.js [hunk 5]
```diff
@@ -52,14 +53,16 @@ const go = async () => {
   console.log("encodedSig", encodedSig);
   console.log("sig length in bytes: ", encodedSig.substring(2).length / 2);
   console.log("dataSigned", dataSigned);
+  let dataSginedBytes = arrayify('0x' + dataSigned)
```
Comment (glitch003) [inline js-sdkTests/signTxn.js:56]: spelling error

**Review by glitch003 — CHANGES_REQUESTED** [review]

**Comment by glitch003** [comment]
let's make this a larger PR to replace the old JS SDK with the new one, in the entire project.  This could just be replacing the import at the top of all the files to use the new SDK.  @joshLong145 can you build on this to accomplish that?

**Comment by johnnynanjiang** [comment]
This PR fixes 2 tests so far

`yarn run hello`
`yarn run sign`

File: js-sdkTests/signTxn.js (hunk 2 above)
Comment (johnnynanjiang) [reply js-sdkTests/signTxn.js:14]: I think there is a issue with encoding large strings of source code for action execution, and @joshLong145 will fix it, in the interim, this is a workaround for the test to pass.

Comment (joshLong145) [reply js-sdkTests/signTxn.js:14]: There is an issue with serializing the provided source to base64 through the `uint8array` package's implementation which causes an `maximum call stack exceeded ` error.  The  the root cause is identified, this is to exercises the signing portion of the tests.

File: js-sdkTests/signTxn.js (hunk 3 above)
Comment (joshLong145) [reply js-sdkTests/signTxn.js:28]: The original auth sig was expired, and throwing invalid signature recovery errors on the node, this was generated to pass the auth as that wasn't the objective of the tests. 

We can make a signature generator for the test register the address to the PKP.

File: js-sdkTests/signTxn.js (hunk 2 above)
Comment (joshLong145) [reply js-sdkTests/signTxn.js]: with the fix to string encoding in the js-sdk [here](https://github.com/LIT-Protocol/js-sdk/commit/0cf5b2310be42a81f9fd247fa702460fe3c619c0) we are now no longer using a cid for testing

**Pull request merged**